File: Lista 7/L7_Q5.c
/*******************************************************************
*   As organizações CSM resolveram dar um aumento de salário aos 
*   seus colaboradores e lhe contrataram para desenvolver o 
*   programa que calcula os reajustes.
*******************************************************************/
#include <stdio.h>

int main(void){
    
    float salario;

    printf("Informe o seu salario: ");
    scanf("%f", &salario);

    if(salario <= 280)
        printf("Salario depois do reajuste eh R$%.2f", salario * 1.20);
    else if (salario <= 700)
        printf("Salario depois do reajuste eh R$%.2f", salario * 1.15);
    else if (salario <= 1500)
        printf("Salario depois do reajuste eh R$%.2f", salario * 1.10);
    else
        printf("Salario depois do reajuste eh R$%.2f", salario * 1.05);
}

File: Período_1/Lista 7/Python/L7_Q5.py
salario = float(input("Informe o salario: "))

if salario <= 280:
    print(f"O salario depois do reajuste é de R${salario + salario * 0.2: .2f}")
elif salario <= 700:
    print(f"O salario depois do reajuste é de R${salario + salario * 0.15: .2f}")
elif salario <= 1500:
    print(f"O salario depois do reajuste é de R${salario + salario * 0.1: .2f}")
else:
    print(f"O salario depois do reajuste é de R${salario + salario * 0.05: .2f}")